01-starting-with-data.Rmd

```diff
@@ -64,6 +64,12 @@ The filename needs to be a character string (or [string](reference.html#string)
 > importing your own data in the future. You can learn more about these
 > options in this supplementary [lesson](01-supp-read-write-csv.html).
 
+> ## Challenge - Reading different decimal point formats {.challenge}
+> Depending on the country you live in, your standard can use the dot or the comma as decimal mark.
+> Also, different devices or software can generate data with different decimal points. 
+> Take a look at '?read.csv' and write the code to load a file called commadec.txt that has numeric values with commas as decimal mark, separated by semicolons.
```

Comment:
Please use back-quotes around `?read.csv` to format as code, and back-quotes around `commadec.txt`.